docs/content/drive.md

```diff
@@ -804,6 +804,18 @@ Keep new head revision of each file forever.
 - Type:        bool
 - Default:     false
 
+#### --drive-size-as-quota
+
+Show storage quota usage for file size.
+
+The storage used by a file is the size of the current version plus any
```

Comment:
This is an auto generated file, and the help here needs to go into the `Help:     "Show storage quota usage for file size.",` line in drive.go

Reply:
Will update that, thanks!